apps/google-analytics-4/lambda/config/serverless-env.dev.yml

```diff
@@ -0,0 +1 @@
+SIGNING_SECRET: IFnWcnKn-oOjWcERvh-GtcU62TotiPDvH59bD7ru2Jwb8BWn4X3iNXsRaWVxI9K0
```

Comment:
Hmmm, this should be excluded via our .gitignore? 🤔

Reply:
ah, damnit! regenerated a fresh one that isn't compromised.